_dev/src/components/FieldRepeater.vue

```diff
@@ -67,7 +67,7 @@ const removeTinyNotifications = () => {
         <Input class="flex-auto" placeholder="Add a color ex: #123456" v-model="f.value" name="bg_color" />
       </div>
     </div>
-    <FileUpload class="my-4" :default="f.default" :path="(f.path) ? f.path : '$/prettyblocks/views/images'"
+    <FileUpload class="my-4" :default="f.default" :path="(f.path) ? f.path : '$/modules/prettyblocks/views/images'"
```

Comment:
```suggestion
    <FileUpload class="my-4" :default="f.default" :path="f.path"
```
I think we don’t need default value here because there is already default value in Vue component.

Reply:
the path is not required, so if there is no path, it should be a defaut path.